crates/gradbench/src/main.rs

```diff
@@ -567,6 +593,33 @@ fn matrix() -> anyhow::Result<()> {
     Ok(())
 }
 
+fn log_command(command: LogCommands) -> anyhow::Result<()> {
```

Comment:
Can you put a docstring on this function, just for consistency with the rest of the code in this file? It can be short.